### PR TITLE
Use icon for theme and background selector

### PR DESCRIPTION
**Pull request by tsauerwein**
... and reduce opacity of map overlay (mobile application).

Closes https://github.com/camptocamp/c2cgeoportal/issues/1950

### CODE REVIEW

File: contribs/gmf/fonts/gmf-icons.svg [hunk 1]
```diff
@@ -1,11 +1,16 @@
 <?xml version="1.0" standalone="no"?>
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" >
 <svg xmlns="http://www.w3.org/2000/svg">
-<metadata>Icons for GeoMapfish</metadata>
+<metadata>Generated by IcoMoon</metadata>
```
Comment (pgiraud) [inline contribs/gmf/fonts/gmf-icons.svg]: Can you please avoid this modification?

**Comment by pgiraud** [comment]
Just one minor comment. Otherwize, it's looks good to me.

File: contribs/gmf/fonts/gmf-icons.svg (hunk 1 above)
Comment (tsauerwein) [inline contribs/gmf/fonts/gmf-icons.svg]: Sure.

**Comment by tsauerwein** [comment]
Thanks for the review.

**Pull request merged**